README.md

```diff
@@ -33,6 +33,64 @@ My Button
 :clipboard: [React UI Documentation](https://github.com/OSCA-Kampala-Chapter/react-ui/tree/main/reactui-docs)
 :office: [Website](https://github.com/OSCA-Kampala-Chapter/react-ui/tree/main/website)
 
+
+## Build Documentation Website
+
+The [website directory](https://github.com/OSCA-Kampala-Chapter/react-ui/tree/main/website) is where we have the documentation that is built on top of [docusaurus v2](https://docusaurus.io/) which uses concepts of [react.js](https://reactjs.org/) so anyone that has done react can just work with it.
+
+
+### Running the development server
+
+To preview your changes as you edit the files, you can run a local development server that will serve your website and reflect the latest changes.
+
+```bash
+cd react-ui
+pnpm run start
+```
+By default, a browser window will open at http://localhost:3000.  
+[pnpm](https://pnpm.io/) is a package manager just like [npm](https://www.npmjs.com/) or [yarn](https://yarnpkg.com/).
+
+Congratulations! You have just created your first Docusaurus site! Browse around the site to see what's available.
+
+### Build
+
+Docusaurus is a modern static website generator so we need to build the website into a directory of static contents and put it on a web server so that it can be viewed. To build the website:
+
+```bash
+pnpm run build
+```
+
+### Updating your Docusaurus version
```

Comment:
@Rishav1707 do we really need this section of updating the docusaurus version? What do you think? I think its not as necessary since its a once in a while thing to update that version